docs/docs/components/reanimated_swipeable.md

```diff
@@ -0,0 +1,219 @@
+
+### Usage:
+
+To use it, import it in the following way:
+
+```js
+import Swipeable from 'react-native-gesture-handler/ReanimatedSwipeable';
+```
+
+## Properties
+
+### `friction`
+
+a number that specifies how much the visual interaction will be delayed compared to the gesture distance. e.g. value of 1 will indicate that the swipeable panel should exactly follow the gesture, 2 means it is going to be two times "slower".
+
+### `leftThreshold`
+
+distance from the left edge at which released panel will animate to the open state (or the open panel will animate into the closed state). By default it's a half of the panel's width.
+
+### `rightThreshold`
+
+distance from the right edge at which released panel will animate to the open state (or the open panel will animate into the closed state). By default it's a half of the panel's width.
+
+### `dragOffsetFromLeftEdge`
+
+distance that the panel must be dragged from the left edge to be considered a swipe. The default value is 10.
+
+### `dragOffsetFromRightEdge`
+
+distance that the panel must be dragged from the right edge to be considered a swipe. The default value is 10.
+
+### `overshootLeft`
+
+a boolean value indicating if the swipeable panel can be pulled further than the left actions panel's width. It is set to `true` by default as long as the left panel render method is present.
+
+### `overshootRight`
+
+a boolean value indicating if the swipeable panel can be pulled further than the right actions panel's width. It is set to `true` by default as long as the right panel render method is present.
+
+### `overshootFriction`
+
+a number that specifies how much the visual interaction will be delayed compared to the gesture distance at overshoot. Default value is 1, it mean no friction, for a native feel, try 8 or above.
+
+### `onSwipeableOpen`
+
+method that is called when action panel gets open (either right or left). Takes swipe direction as
+an argument.
+
+### `onSwipeableClose`
+
+method that is called when action panel is closed. Takes swipe direction as
+an argument.
+
+### `onSwipeableWillOpen`
+
+method that is called when action panel starts animating on open (either right or left). Takes swipe direction as
+an argument.
+
+### `onSwipeableWillClose`
+
+method that is called when action panel starts animating on close. Takes swipe direction as
+an argument.
+
+### `renderLeftActions`
+
+method that is expected to return an action panel that is going to be revealed from the left side when user swipes right.
+This map describes the values to use as inputRange for extra interpolation:
+
+SharedValue: [startValue, endValue]
+
+progress: [0, 1]
+
+drag: [0, +]
```

Comment:
I don't understand the `[0, +]`, `[0, -]` notation

Reply:
It means these values travel from `0` to `positive infinity` and from `0` to `negative infinity` respectively.
I copied this notation from the legacy `Swipeable` docs for consistency, but i can write it down differently if you'd prefer.

Reply:
`[0, +]` is not that bad, but `[0, -]` is confusing

Reply:
is `[0, +∞]` and `[0, -∞]` fine?

Reply:
What I meant is not the infinity sign, but rather the notation of `[0, -∞]` (it doesn't follow formal definition of interval). I'm not sure though if that is what @kacperkapusciak meant.

Reply:
Hey so what's the verdict on this issue?

As far as I see, the formal way would be to note it this way: `(-∞, 0]` and `[0, +∞)`
But since here, it represents a mapping from `input` to `output`, i think `[0, -∞)` and `[0, +∞)` would be more readable.

Please let me know, which one you think would work better @m-bert.

Reply:
Well, I don't like `[0, -∞)` because it looks very unnatural to me. On the other hand I cannot think of any better idea 😞  One could argue that if we only talk about input and output range, we could leave formal notation (but it is not a strong argument)

Anyway, it was @kacperkapusciak who has started this discussion and I want to know what's his stance on this